src/utils/buildContext.ts

```diff
@@ -1,41 +1,73 @@
+import { ProfileFormValues } from '../components/composite/forms/ProfileForm';
+import { ProfileFormValuesType } from '../components/composite/forms/ProfileFormWithContext';
+import { formatDatabaseLocation } from '../domain/location/LocationUtils';
 import {
   CreateContextInput,
   CreateReferenceInput,
   Reference,
+  Location,
   UpdateContextInput,
   UpdateReferenceInput,
 } from '../models/graphql-schema';
 
-interface ContextObject {
+export interface ContextObject {
   background?: string;
   impact?: string;
   references?: Reference[];
   tagline?: string;
+  location?: Location;
   vision?: string;
   who?: string;
 }
 
+export const mapProfileToContext = (values: ProfileFormValues) => {
```

Comment:
you are not mapping profile to context. Profile does not have most of the fields in `values`.
`name`, `nameID` are not even members of `ContextObject`.
You have something mixed up here.